test/staking/Integration.ts

```diff
@@ -505,7 +525,10 @@ describe("Integration", function () {
       );
       // console.log(delegatorRewardsRecevied.map(a => a.toString()));
       expect(delegatorRewardsRecevied.every((a) => a.gt(0))).to.be.true;
-      expect(delegatorRewardsRecevied.every((a) => a.eq(delegatorRewardsRecevied[0]))).to.be.true;
+      for (let index = 0; index < delegatorRewardsRecevied.length - 1; index++) {
+        const element = delegatorRewardsRecevied[index];
+        expect(element).to.be.closeTo(delegatorRewardsRecevied[index + 1], delegatorRewardsRecevied[index + 1].div(1000))
```

Comment:
Why close to, it should be based on whatever tickets were given to each na, sure probably not everyone has equal tickets, then the distribution should be based on ticket ratios right ?

Reply:
In the given test_scenario, every everyone has equal tickets.